functions/snippets/src/test/java/com/example/functions/ImageMagickTests.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.functions;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.testing.TestLogHandler;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+
+public class ImageMagickTests {
+  private static String BUCKET_NAME = System.getenv("FUNCTIONS_BUCKET");
+  private static String BLURRED_BUCKET_NAME = System.getenv("BLURRED_BUCKET_NAME");
```

Comment:
These can be pre-existing buckets.

Reply:
and well known.

Reply:
`BUCKET_NAME` now points to `nodejs-docs-samples-tests`.